lite2/client.go

```diff
@@ -675,7 +675,7 @@ func (c *Client) bisection(
 		case ErrNewValSetCantBeTrusted:
 			// do add another header to the end of the cache
 			if depth == len(headerCache)-1 {
-				pivotHeight := (headerCache[depth].sh.Height + trustedHeader.Height) / 2
+				pivotHeight := (headerCache[depth].sh.Height + trustedHeader.Height) * 9 / 16
```

Comment:
magic const

Reply:
there is some rationale behind the madness

Reply:
could you please extract it into a const and add a comment?

Reply:
sure :+1: